src/tests/UnitTests/Navigation/WPF/NavigationServiceTests.registration.cs

```diff
@@ -0,0 +1,95 @@
+﻿using CodeMonkeys.Navigation;
+using CodeMonkeys.Navigation.WPF;
+using CodeMonkeys.UnitTests.Navigation.ViewModels;
+using CodeMonkeys.UnitTests.Navigation.WPF.Views;
+
+using NUnit.Framework;
+
+using System.Linq;
+
+namespace CodeMonkeys.UnitTests.Navigation.WPF
+{
+    public partial class NavigationServiceTests
+    {
+        [Test]
+        public void Register_WhenThereIsNoRegistrationWithSameTypes_AddsRegistration()
+        {
+            int expectedRegistrationsCount = 3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'expectedRegistrationsCount'.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)